### ISSUE TITLE
Feature Request: a to-do checklist

### ISSUE THREAD

**Issue by Aevann** [turtl/js#209]

**Comment by tdbe** [comment]
+1 to this. The main reason to use a Notes app is to write (To Do) Lists of things. And to check off those you've completed. And maybe to colour or grey them out by importance as a nice-to-have.

**Comment by JulienPalard** [comment]
As the apps uses markdown, if you use the `- [ ]`  syntax, it will create a checkable box, you can also pre-check it by using the `- [x]` syntax, like

- [x] buy this
- [x] buy that
- [ ] eat this
- [ ] drink that

**Comment by Aevann** [comment]
- [ ] doesn't work on the android app

**Comment by JulienPalard** [comment]
Exact, only works on the web interface :( This issue should be renamed to "make `- [x]` work on Android" þ

**Comment by holocronweaver** [comment]
Note taking apps are about speed and ease. Typing lists via Markdown is too cumbersome, especially on mobile. Something more streamlined like what Google Keep does is preferable, even on desktop.

**Comment by mr-kojo** [comment]
` - [x]` markdown does not work on the Windows client either.

**Comment by koulaxizis** [comment]
I'd love to see that!!
